.travis.yml

```diff
@@ -15,6 +17,5 @@ install:
  - npm install
 
 before_install:
- - npm update -q  
- - npm install -g grunt-cli
```

Comment:
Note that removing grunt has also impact in documentation. For instance: https://github.com/telefonicaid/iotagent-node-lib/tree/master/doc/usermanual.md#development-documentation

Please, do a grep for grunt in .md files in order to find them and fix.

Reply:
Fixed with d247ceaac4299a39f8319693b23b606e33da824a